plugins/dbviewer/frontend/public/javascripts/countly.views.js

```diff
@@ -489,6 +489,12 @@
                 backToDBViewer: function() {
                     window.location = '#/manage/db/' + this.db + '/' + this.collection;
                 },
+                decode: function(value) {
+                    if (typeof value === "string") {
+                        return value.replace(/^&#36;/g, "$").replace(/&#46;/g, '.').replace(/&lt;/g, '<').replace(/&gt;/g, '>').replace(/&le;/g, '<=').replace(/&ge;/g, '>=').replace(/&quot;/g, '"').replace(/&#39;/g, "'");
+                    }
+                    return value;
+                },
```

Comment:
Please use this:
https://github.com/Countly/countly-server/blob/master/frontend/express/public/javascripts/countly/countly.common.js#L356